packages/@recuperateur/async-busboy/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@recuperateur/async-busboy",
+  "version": "1.2.1",
+  "description": "Promise based multipart form parser",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Recuperateur/async-busboy"
```

Comment:
That link will not be maintained.
The README should be included in the directory.
Maybe the examples too.
The whole package in fact (with its LICENSE, and tests).